frontend/src/router/router.js

```diff
@@ -52,6 +52,12 @@ const routes = [
         meta: { nav: 'product' },
         component: () => import('../views/ProductDetails.vue')
     },
+    {
+        path: '/products/:id/edit-stock',
+        name: 'edit-stock',
+        meta: { nav: 'edit-stock' },
```

Comment:
De nav variable bepaald welk item in de navbar gekleurt word. Denk dat dit bij het product hoort dus dat dit nav: 'product' kan zijn. Anders is de gehele onderste balk zwart